mls-rs-crypto-webcrypto/src/key_type.rs

```diff
@@ -0,0 +1,106 @@
+use js_sys::{Array, Reflect, Uint8Array};
+use wasm_bindgen::JsValue;
+use wasm_bindgen_futures::JsFuture;
+use web_sys::{CryptoKey, CryptoKeyPair, EcKeyImportParams, HmacImportParams, SubtleCrypto};
+
+#[derive(Clone, Debug)]
+pub(crate) enum KeyType {
+    Hmac(&'static str),
+    AesGcmE,
```

Comment:
I assume this is AesGcmEncrypt / Decrypt? Probably better to make that clear

Reply:
Fixed